src/main/java/com/exasol/adapter/dynamodb/remotetablequery/normalizer/DnfNormalizer.java

```diff
@@ -0,0 +1,34 @@
+package com.exasol.adapter.dynamodb.remotetablequery.normalizer;
+
+import org.logicng.formulas.Formula;
+import org.logicng.transformations.dnf.DNFFactorization;
+
+import com.exasol.adapter.dynamodb.remotetablequery.QueryPredicate;
+
+/**
+ * This class normalizes a {@link QueryPredicate} structure to the disjunctive normal form.
+ */
+@java.lang.SuppressWarnings("squid:S119") // DocumentVisitorType does not fit naming conventions.
+public class DnfNormalizer<DocumentVisitorType> {
+    private final QueryPredicateToLogicngConverter<DocumentVisitorType> converter;
+
+    /**
+     * Creates an instance of {@link DnfNormalizer}
```

Comment:
```suggestion
     * Create an instance of {@link DnfNormalizer}.
```

Reply:
I think we usually describe methods from `you` side: create, get, add etc. 
It not a mistake in your case, it's just nice to have them unified in our projects

Reply:
ok. I tried to change it everywhere.